src/app/i18n/locales/en.json

```diff
@@ -27,6 +27,26 @@
     },
     "cta": "Get the deal"
   },
+  "notFound": {
+    "header": "Error 404",
+    "title": "Nothing to see here...",
+    "description": "The page you are looking for could not be found.",
+    "card1": {
+      "title": "Return Home",
+      "description": "Go back to Internxt Drive.",
+      "urlRedirect": "https://drive.internxt.com/app"

```

Comment:
Not add url to translation files. Use the navigationService in the cards object that has you created to redirect the user to other view of the web, use constants or what do you think fits better depend of the case.